projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.css

```diff
@@ -1,10 +1,17 @@
 ::ng-deep .ngx-location-picker-marker {
-    width: 20px;
-    height: 20px;
-    line-height: 20px;
-    text-align: center;
-    border-radius: 50%;
-    display: block;
+    position: relative;
+    top: -36px;
+    left: -5px;
+    font-size: 40px;
+}
+
+::ng-deep .is-picking .leaflet-grab {
```

Comment:
core-branding-gewijs werken we normaal gezien met de font awesome icons, bvb deze: https://fontawesome.com/icons/map-pin?style=solid

Reply:
added suggested marker